src/components/AuthenticatedAppHeader/index.js

```diff
@@ -142,6 +142,7 @@ export default function AppHeader() {
             refreshNotifications={refreshNotifications}
             shouldOpen={shouldOpenNotificationPane}
             setShouldOpen={setShouldOpenNotificationPane}
+            currentUserGuid={meData?.guid}
```

Comment:
I fluctuated a little bit about whether I wanted this to be a prop here or just derive it with an api call in the child component.

Reply:
I ran into the same question on #464. I passed it as a prop too. It looks like React Query recommends just deriving it though.
https://github.com/TanStack/query/discussions/463

Reply:
I kept it as a prop for this ticket...

Reply:
That seems like a good idea to me. It is something we can look into and try to implement going forward.